In [4]:
from scipy.stats import norm, ttest_1samp

In [3]:
data = norm.rvs(loc=2, scale=30, size=200)
data

array([ 17.83121897,  19.51119177,  -4.47115369,  15.25144097,
        25.51467099, -37.35662734, -17.21150635,  16.31515966,
        48.87416013, -15.49271696,  57.20397291,  38.09411202,
        18.93168222,   8.83741152,   5.25073547,   9.96445753,
       -10.14471462,   8.37251606, -12.66765119,  31.88364813,
        64.81376947,  56.15848554,  -0.93766436, -10.26045355,
        40.49078691,  38.50556399,  -9.23540023,  32.29215962,
        25.78802063,  -4.10665713,   7.40799793, -23.4071895 ,
       -34.51916373, -18.12209641,  11.65317854, -14.88365252,
        62.11673301, -12.90333626, -41.58937634, -22.96377066,
         0.18657886,  39.11751155,  68.44150952, -45.14466174,
       -11.53719709, -38.42094276,  46.61164744,   4.96446362,
       -26.12344661, -38.11681686,  -2.44783955,  13.51494513,
       -51.0474655 , -69.66656452,  21.71146528, -48.87058642,
       -76.61413125,   9.5847381 ,  11.31909756, -13.97825054,
       -53.8595928 , -40.83972185,  39.82273529,   7.41

In [5]:
ttest_1samp(data, 2.5)

Ttest_1sampResult(statistic=0.5855568989880374, pvalue=0.5588367713642137)

p-value wynosi ~0.56. Nie ma więc podstaw do odrzucenia hipotezy zerowej, mówiącej, że średnia tego rozkładu jest równa 2.5